data/spells/lib/spells.lua

```diff
@@ -258,6 +258,16 @@ RANGE = {
 [16] = {1, 2},   [17] = {1, 2},   [18] = {1, 2}, [19] = {1, 2}
 }
 
+COORDINATES_AREA_CIRCLE5X5 = {
+{x = 0, y = 0},
+{x = 1, y = 0}, {x = 2, y = 0}, {x = 3, y = 0}, {x = 4, y = 0}, {x = 5, y = 0},
```

Comment:
Well you can atleast clean the table up abit.
As I told you before, making it shorter isen't always better.

We have to keep in mind that people might have to change your code at some time and it will help ALOT if it's clean and easy to read, if I found a bug and tracked it down to this table I would just scrap it because of how messy it is, creating more work for no reason.